### DISCUSSION TITLE
Specific Field Name "RD_ID_1" in Data Fields cannot be found

### DISCUSSION THREAD

**Discussion by JoeChen7702** [ldproxy/ldproxy#1025] [API]
Related to [#1021](https://github.com/interactive-instruments/ldproxy/issues/1021)

We built the LDProxy in our internal network and developed an additional API as proxy. In theory it will call our API, and the API will send request to LDProxy API. Below is the sample of the call,

We have the same question as #1021. We got the response below after calling LDProxy,

`{
  "error": {
    "status": 400,
    "title": "Bad Request",
    "detail": "Property 'RD_ID_1' was not found"
  }
}
`
Can anyone advise how to fix or modify?

**Comment by cportele** [comment]
We do not have sufficient information to provide answers. We cannot access the API (https://iapi.csdidev.gov.hk/apim/td_rcd_1638949160594_2844 cannot be accessed) and have no information about the API configuration.
